e2e/cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need this file or can we delete it?

Reply:
This one, yes.